components/Blocks/PlainHero.css

```diff
@@ -0,0 +1,30 @@
+.root {
+  background: var(--color-offwhite);
+  color: var(--color-grey);
```

Comment:
I'm just not sure it makes anything better or easier. At some point you'd have to make a distinction between foreground and background, then naming the variable something that was understandable and yet not too verbose... I'm just not convinced it's actually better.